doc/source/whatsnew/v0.21.1.txt

```diff
@@ -7,6 +7,45 @@ This is a minor release from 0.21.1 and includes a number of deprecations, new
 features, enhancements, and performance improvements along with a large number
 of bug fixes. We recommend that all users upgrade to this version.
 
+.. _whatsnew_0211.special:
+
+Restore Matplotlib datetime Converter Registration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Pandas implements some matplotlib converters for nicely formatting the axis
+labels on plots with ``datetime`` or ``Period`` values. Prior to pandas 0.21.0,
+these were implicitly registered with matplotlib, as a side effect of
+``import pandas``. In pandas 0.21.0, we required users to explicitly register
+the converter.
+
+.. code-block:: python
+
+   >>> from pandas.tseries import converter
+   >>> converter.register()
+
+This caused problems for some users, so we're temporarily reverting that change;
+pandas will again register the converters on import. Using the converters
+without explicitly registering the formatters will cause a ``FutureWarning``:
```

Comment:
I would not use 'the formatters' as it actually points to the same as 'converters' but with another word, just making it confusing. I would either just leave it out or use 'them' to avoid repeating converters

Reply:
Meant to use formatters everywhere. Fixing.